Formula/n98-magerun.rb

```diff
@@ -0,0 +1,15 @@
+class N98Magerun < Formula
+  desc "The swiss army knife for Magento developers, sysadmins and devops."
+  homepage "http://magerun.net/"
+  url "https://files.magerun.net/n98-magerun.phar"
+  version "1.0.0"
```

Comment:
Can you comment on how you came up with this URL and version? I can see a lot of releases https://files.magerun.net/ and it doesn't look like 1.0.0 is the latest version. Actually, it seems the latest version is 1.97.22 and we'll need to reference that file explicitly. Otherwise installation will break if the file referenced by the current URL will be updated and the checksum no longer matches.